docs/getting-started/installing-the-library-in-an-existing-application.md

```diff
@@ -0,0 +1,35 @@
+# Installing the library in an existing application
+
+If your existing application has a web server to run the app locally, you can install the US Forms System Library in that app.
+
+### In this guide
+
+- [Install the library and dependencies](#install-the-library-and-dependencies)
+- [Create required files](#create-required-files)
+
+### Install the library and dependencies
+
+1. Install the library:
+```bash
+npm install --save https://github.com/usds/us-forms-system.git
```

Comment:
Once we move to publish this library as an npm package, we'll have to update these instructions. Need to remind myself to do this at that point!